src/api/postFile.ts

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios';
+import { url } from '../shared/variable.shared';
+
+
+const changeFileToFormData = (file: File): FormData => {
+    const formData = new FormData();
+    formData.append('file', file, file.name);
+    return formData
+}
+
+
+const sendFile = async (file: File): Promise<boolean> => {
+    const formData = changeFileToFormData(file);
+    let result = ''
+    if (url)
+    try{
+        await axios.post(url+'/files', formData, { headers: {
+            'Content-Type': `multipart/form-data`
+        } })
+        .then((res) => {
```

Comment:
jak robisz try catcha to nie rob takiego chainingu

Reply:
zrob np.
``` 
try {
await axios.pos()
return true
} catch (e) { ... }
return false
```

Reply:
w sumei uzycie true jak sie uda byloby chyba logiczniejsze, bo teraz w logice masz chyba na odwrot

Reply:
zmieniłem na true/false i usunałem chaining